api/src/main/java/org/openmrs/api/ProviderRoleInUseException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api;
+
+/**
+ * Thrown when user attempts to delete a provider role that is currently linked to a provider
+ */
+public class ProviderRoleInUseException extends Exception {
+	
```

Comment:
Do we need a dedicated class for this when we already have `CannotUpdateObjectInUseException`?